SpatialGDK/Source/SpatialGDK/Public/Utils/SchemaDatabase.h

```diff
@@ -95,6 +95,15 @@ class SPATIALGDK_API USchemaDatabase : public UDataAsset
 	UPROPERTY(Category = "SpatialGDK", VisibleAnywhere)
 	TMap<uint32, FString> ComponentIdToClassPath;
 
+	UPROPERTY(Category = "SpatialGDK", VisibleAnywhere)
```

Comment:
Nit: I feel like these should be combined into a `TMap<ESchemaComponentType, TSet<uint32>> ComponentIds;`

If we add a new component type then we would have to add a whole new variable here.

Reply:
Oh wait this might not be possible due to UPROPERTY shenanigans. Worth trying.

Reply:
`nested containers are not supported.` 😢